pop-cli-for-appchains/guides/calling-a-chain.md

```diff
@@ -0,0 +1,72 @@
+# Call a chain
+
+### Interactive Guidance (Recommended)
+
+Interact with a chain **using** Pop CLI's interactive guidance by simply entering:
+
+```shell
+pop call chain
+```
+
+You will be prompted to select a pallet, choose a dispatchable function, specify any required arguments, then provide connection details and the information of the account that will sign the extrinsic.
```

Comment:
We should be careful with using extrinsic in stead of transaction, the latter is easier to understand for most people.
```suggestion
You will be prompted to select a pallet, the dispatchable function and its required arguments, connection details for the chain, and the account to sign the transaction.
```

Reply:
https://github.com/r0gue-io/pop-docs/pull/12/commits/0447185c620cce672862cfcdc515396c75e85671